components/central-logger/org.wso2.carbon.identity.central.log.mgt/src/main/java/org/wso2/carbon/identity/central/log/mgt/utils/LoggerUtils.java

```diff
@@ -57,45 +56,30 @@ public class LoggerUtils {
     private static final Log log = LogFactory.getLog(LoggerUtils.class);
     private static final String CORRELATION_ID_MDC = "Correlation-ID";
     private static final String FLOW_ID_MDC = "Flow-ID";
-    private static final String CLIENT_COMPONENT = "clientComponent";
 
-   /**
+    /**
     * Config value related to masking sensitive information from logs.
     */
     public static boolean isLogMaskingEnable;
 
     /**
-     * @param initiatorId   Request initiator's id.
-     * @param initiatorName Request initiator's name.
-     * @param initiatorType Request initiator's type.
-     * @param evenType      State changing event name.
-     * @param targetId      Target resource's id.
-     * @param targetName    Target resource's name.
-     * @param targetType    Target resource type.
-     * @param dataChange    Changing data.
+     * This method is used to trigger audit log event
+     *
+     * @param auditLogBuilder  Audit log builder
+     * @param isLoggingEnabled Is new audit logging enabled in the component
      */
-    public static void triggerAuditLogEvent(String initiatorId, String initiatorName, String initiatorType,
-                                            String evenType, String targetId, String targetName, String targetType,
-                                            String dataChange) {
+    public static void triggerAuditLogEvent(AuditLog.AuditLogBuilder auditLogBuilder, boolean isLoggingEnabled) {
```

Comment:
do we need this isLoggingEnabled param?

Reply:
This is to makesure that new set of audit logging is enabled in the component level. There could be chances that the developer might miss to check them in the component level. That is why I added this check here.